docs/src/main/asciidoc/kubernetes.adoc

```diff
@@ -8,7 +8,7 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 include::./attributes.adoc[]
 
 This guide covers generating and deploying Kubernetes resources based on sane defaults and user supplied configuration.
-In detail, it supports generating resources for <<Kubernetes>>, <<OpenShift>> and <<Knative>>. Also it supports automatically <<Deployment>> these resources to the target platform.
+In detail, it supports generating resources for <<Kubernetes>>, <<OpenShift>> and <<Knative>>. Also it supports automatic deployment of these resources to the target platform.
```

Comment:
I prefer we avoid having links without ids. If we change the titles, they get broken.

Reply:
I can't say I know much about adoc, but I think I fixed it now?

Reply:
The whole point of it is to define ids on the paragraphs so that you refer to fixed ids

Reply:
Gotcha, thanks (as you already know, I never developed a taste for docs 😂). I'll fix it tomorrow morning

Reply:
Was able to fix it quickly. I assume it's correct now?